CONTRIBUTING.md

```diff
@@ -1,186 +1,157 @@
+The rest of this document describes the technical details of getting set up to develop, and make your first contribution to Mitiq.
 
-### Version control with git
-git is a language that helps keeping track of the changes made. Have a look at these guidelines for getting started with [git workflow](https://www.asmeurer.com/git-workflow/).
-Use short and explanatory comments to document the changes with frequent commits.
 
-### Forking the repository
-You can fork mitiq from the github repository, so that your changes are applied with respect to the current master branch. Use the Fork button, and then use git from the command line to clone your fork of the repository locally on your machine.
-```bash
-(base) git clone https://github.com/your_github_username/mitiq.git
-```
-You can also use SSH instead of a HTTPS protocol.
+## Development environment
 
-### Working in a virtual environment
-It is best to set up a clean environment with anaconda, to keep track of all installed applications.
-```bash
-(base) conda create -n myenv python=3
+1. Ensure you have python 3.8 or greater installed. If not, you can find the downloads [here](https://www.python.org/downloads/). 
+2. Set up a virtual environment to isolate dependencies. This can be done with many different tools including [Virtualenv](https://virtualenv.pypa.io/en/latest/), [Pipenv](https://pypi.org/project/pipenv/), [Poetry](https://python-poetry.org/), and [Anaconda](https://www.anaconda.com/products/distribution). In what follows we will use Anaconda, but if you're familiar with other tools feel free to use those.
+3. Set up a local version of the [Mitiq repository](https://github.com/unitaryfund/mitiq). To do this you will need to use `git` which is a version control system. If you're unfamiliar, check out the [docs](https://git-scm.com/), and learn about what the typical [`git` workflow](https://www.asmeurer.com/git-workflow/) looks like.
+4. Inside the Mitiq repository (`cd mitiq`), activate a virtual environment. With conda this is done using the following command.
 ```
-accept the configuration ([y]) and switch to the environment
-```bash
-(base) conda activate myenv
-(myenv) conda install pip
+conda create --name myenv python=3
+conda activate myenv
 ```
-Once you will finish the modifications, you can deactivate the environment with
-```bash
-(myenv) conda deactivate myenv
+5. Install the dependencies. First, to get an updated version of [`pip`](https://pypi.org/project/pip/) inside the virtual environment run `conda install pip` followed by
 ```
-
-### Development install
-In order to install all the libraries useful for contributing to the
-development of the library, from your local clone of the fork, run
-
-```bash
-(myenv) pip install -e .
-(myenv) pip install -r dev_requirements.txt
+pip install -e .
+pip install -r requirements.txt
```

Comment:
```suggestion
pip install -e .[development]
```
This is the command that we use in CI to install Mitiq together with all dev dependencies.  
What is now written here is also good, but I guess `requirements.txt` should be `dev_requirements.txt`